include/dt-bindings/pinctrl/stm32-pinctrl.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _STM32_PINCTRL_H_
+#define _STM32_PINCTRL_H_
+
+#include <dt-bindings/pinctrl/stm32-pinctrl-common.h>
+/**
+ * @brief PIN configuration bitfield
+ *
+ * Pin configuration is coded with the following
+ * fields
+ *    Alternate Function  [ 0 : 7 ]
+ *    GPIO Mode           [ 8 : 9 ]
+ *    GPIO Output type    [ 10 ]
+ *    GPIO Speed          [ 11 : 12 ]
+ *    GPIO PUPD config    [ 13 : 14 ]
+ *
+ * Applicable to STM32F3, STM32F4, STM32L4 series
+ */
+
+#define STM32_AFR_MASK			0xF
```

Comment:
This is "Alternate function" [0,7]
This was already present and I removed the AFR_SHIFT in clean up since not used.
Maybe it could be made clearer.
Is this blocking? I'll have trouble to update this PR soon, but I would be pleased to make it clearer later on.